file_reader_code.R

```diff
@@ -0,0 +1,66 @@
+if (!require("pacman")) install.packages("pacman")
```

Comment:
Don't force other people to install packages, this should just be a library call